SonarQube.MSBuild.Tasks/WriteProjectInfoFile.cs

```diff
@@ -50,7 +50,7 @@ public class WriteProjectInfoFile : Task
         #region Constructors
 
         public WriteProjectInfoFile()
-            : this(new EncodingProvider())
+            : this(new Common.EncodingProvider())
```

Comment:
Is this needed?

Reply:
Yes. After the refrerence changes it the build fails with name clash.